README.md

```diff
@@ -25,30 +25,3 @@ i18n
     }
   });
 ```
-
```

Comment:
perhaps we should mention internal caching now, so that users don't need to actually use `i18nextLocalStorageBackend`?

Reply:
Yeah, I think we should also make the expiration time configurable for e.g. testing. I'll add this in a follow-up PR